diffsims/__init__.py

```diff
@@ -22,12 +22,16 @@
 
 import numpy as np
 
-from .generators.diffraction_generator import DiffractionGenerator
+from .generators.diffraction_generator import DiffractionGenerator, AtomicDiffractionGenerator
 from .generators.library_generator import DiffractionLibraryGenerator
 from .generators.library_generator import VectorLibraryGenerator
 
 from .sims.diffraction_simulation import DiffractionSimulation
 
+from .utils.probe_utils import probeFunction, BesselProbe
+from .utils.fourier_transform import toRecip, fromRecip, getRecipPoints, getDFT
+from .utils.discretise_utils import getDiscretisation
+
```

Comment:
For all 3 of the above lines - we try to keep to lower case for functions and use only capitals like that in class names. This is just a code style thing, but consistency on that is helpful.